packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts

```diff
@@ -0,0 +1,159 @@
+import { deploySponsorship } from "./deploySponsorshipContract"
+import { MaintainOperatorValueService } from "../../../../src/plugins/operator/MaintainOperatorValueService"
+import { OperatorServiceConfig } from "../../../../src/plugins/operator/OperatorPlugin"
+
+const config = Chains.load()["dev1"]
+const adminPrivKey = "0x4059de411f15511a85ce332e7a428f36492ab4e87c7830099dadbf130f1896ae"
+const theGraphUrl = `http://${process.env.STREAMR_DOCKER_DEV_HOST}:8000/subgraphs/name/streamr-dev/network-subgraphs`
+
+const logger = new Logger(module)
+
+jest.setTimeout(60 * 1000)
+
+describe("MaintainOperatorValueService", () => {
+    const chainURL = config.rpcEndpoints[0].url
+
+    let provider: Provider
+    let operatorWallet: Wallet
+    let operatorContract: Operator
+    let token: IERC677
+    let adminWallet: Wallet
+    let streamId1: string
+    let streamId2: string
+
+    let opertatorConfig: OperatorServiceConfig
+
+    const deployNewOperator = async () => {
+        const operatorWallet = Wallet.createRandom().connect(provider)
+        logger.debug("Funding", { address: operatorWallet.address })
+        await (await token.transfer(operatorWallet.address, parseEther("1000"))).wait()
+        await (await adminWallet.sendTransaction({
+            to: operatorWallet.address,
+            value: parseEther("1")
+        })).wait()
+
+        logger.debug("Deploying operator contract")
+        const operatorContract = await deployOperatorContract(config, operatorWallet)
+        logger.debug(`Operator deployed at ${operatorContract.address}`)
+        opertatorConfig = {
+            operatorContractAddress: operatorContract.address,
+            provider,
+            theGraphUrl,
+            fetch,
+            signer: operatorWallet
+        }
+        return { operatorWallet, operatorContract }
+    }
+
+    beforeEach(async () => {
+        provider = new JsonRpcProvider(chainURL)
+        logger.debug("Connected to: ", await provider.getNetwork())
+
+        adminWallet = new Wallet(adminPrivKey, provider)
+
+        token = new Contract(config.contracts.LINK, tokenABI, adminWallet) as unknown as IERC677
+        const timeString = (new Date()).getTime().toString()
+        const streamPath1 = "/operatorvalueservicetest-1-" + timeString
+        const streamPath2 = "/operatorvalueservicetest-2-" + timeString
+        streamId1 = adminWallet.address.toLowerCase() + streamPath1
+        streamId2 = adminWallet.address.toLowerCase() + streamPath2
+        const streamRegistry = new Contract(config.contracts.StreamRegistry, streamRegistryABI, adminWallet) as unknown as StreamRegistry
+        logger.debug(`creating stream with streamId1 ${streamId1}`)
+        await (await streamRegistry.createStream(streamPath1, "metadata")).wait()
+        logger.debug(`creating stream with streamId2 ${streamId2}`)
+        await (await streamRegistry.createStream(streamPath2, "metadata")).wait();
```

Comment:
We could use StreamrClient to create the streams:

```suggestion
        const client = new StreamrClient({
            auth: {
                privateKey: adminWallet.address
            }
        })
        stream1 = await client.createStream(`/operatorvalueservicetest-1-${Date.now()}`)
        stream2 = await client.createStream(`/operatorvalueservicetest-2-${Date.now()}`)
        await client.destroy()
```

Reply:
Changed